js/base/Exchange.js

```diff
@@ -2746,4 +2746,10 @@ module.exports = class Exchange {
         }
         return [ marginMode, params ];
     }
+
+    arrayLength (arr) {
```

Comment:
this method was needed across multiple tests. I know your concern about this, so, this is not used inside derived exchange classes (to cause confusion for regular users) but we need it extensively in tests because of mutli-language transpilation